apps/web/src/services/asset-inventory/collector/modules/CollectorScheduleForm.vue

```diff
@@ -43,43 +43,86 @@
 </template>
 
 <script lang="ts" setup>
-import { defineProps, reactive, computed } from 'vue';
+import type { ComputedRef } from 'vue';
+import {
+    defineProps, defineEmits, reactive, computed,
+} from 'vue';
 
 import {
     PFieldGroup, PButton, PFieldTitle, PToggleButton,
 } from '@spaceone/design-system';
-import type { Dayjs } from 'dayjs';
 import dayjs from 'dayjs';
 import { range, size } from 'lodash';
 
 import { store } from '@/store';
 
-interface ScheduleHours {
-    [time: string]: Dayjs;
-}
-
 const props = defineProps<{
     editHours: boolean;
+    utcHours?: string[];
 }>();
 
-const hoursMatrix = range(24);
-const state = reactive({
-    timezone: computed(() => store.state.user.timezone),
+const emits = defineEmits<{(event: 'update:hours', value: string[]): void;
+}>();
+
+const hoursMatrix: string[] = range(24).map((hour) => hour.toString());
+const selectedUtcHoursSet = new Set<string>();
+const state = reactive<{
+    timezone: ComputedRef<string>;
+    isAutoSchedule: boolean;
+    isAllHoursSelected: ComputedRef<boolean>;
+    selectedUtcHours: string[];
+    timezoneAppliedHours: ComputedRef<string[]>;
+}>({
+    timezone: computed<string>(() => store.state.user.timezone),
```

Comment:
Question: Why did you rewrite the type, even though you wrote it on the reactive generics?

Reply:
Because state is regarded as `any` type in my IDE... I had just tried!